pingcap/tiflow
# server(cdc): avoid printing help messages when cdc server exits

### What problem does this PR solve?


Issue Number: close #5519

### What is changed and how it works?
In `func RunE()` of the `cdc server` command, if no error occurs in the validation, return nil directly in spite of errors thrown by the execution. Handle the execution errors using `cobra.CheckErr()`

#### Tests 

 - Integration test 
   - `tests/integration_tests/cdc_server_tips`: The tests cover two scenarios: one `cdc server` command with invalid arguments should print usage tips, while the other is validated and should not.
 - Manual test
   - Start a tidb cluster and add a cluster using `cdc server`
   - Create a changefeed task using `cli changefeed`
   - Press `Ctrl + C` and in the terminal of `cdc server` and check whether there are prompt messages printed
 
##### Will it cause performance regression or break compatibility?
No
##### Do you need to update user documentation, design documentation or monitoring documentation?
No
### Release note 

```release-note
Clear redundant cdc server prompt in a normal exit
```
